src/gc/gc.d

```diff
@@ -1876,13 +1921,13 @@ struct Gcx
             }
         }
 
+        mappedPages += npages;
+        lowMem = isLowOnMem(mappedPages * PAGESIZE);
```

Comment:
Wouldn't it be better to call `isLowOnMem` whenever the decision for allocating a new pool has to be made instead of caching it from the previous call to `newPool`? The former is seldom enough, slightly simpler and might take other changes outside the GC into account.